src/scraper/__init__.py

```diff
@@ -97,32 +215,62 @@ def get_class_and_method(self, county):
             module = importlib.import_module(module_name)
             
             # Retrieve the class from the module
-            cls = getattr(module, class_name)
+            cls = getattr(module, class_name, None)
             if cls is None:
-                print(f"Class '{class_name}' not found in module '{module_name}'.")
-                return None, None
-            
+                raise AttributeError(f"Class '{class_name}' not found in module '{module_name}'")
```

Comment:
Good use of exceptions, this is much cleaner raising for bad input rather than returning None IMO